src/Dropdown/index.tsx

```diff
@@ -172,7 +176,7 @@ const Dropdown: IDropdown = ({
   }
 
   return (
-    <Wrapper>
+    <Wrapper minHeight={props.minHeight} minWidth={props.minWidth}>
```

Comment:
It would be nice to avoid a wrapper or W/H = 100% because in my world I would expect input (DropDown in this case) to be controller by the parent component if it makes sense. In this instance it could be just a fragment - if needed.